### ISSUE TITLE
No Transparency in any sidebars (Finder, App Store, etc.)

### ISSUE THREAD

**Issue by thegurgz** [SpiraMira/HybridMode-Public#6]
I am not getting any transparency using the Hybrid mode (HIToolBox+CoreUI).
Light Finder with CoreUI file:
![screen shot 2018-11-02 at 2 53 57 pm](https://user-images.githubusercontent.com/44713744/47958296-edea4d80-df85-11e8-8353-baab5dcd4f1b.png)
Dark Finder with HIToolBox file:
![screen shot 2018-11-03 at 4 33 07 pm](https://user-images.githubusercontent.com/44713744/47958311-2e49cb80-df86-11e8-9313-5f09361e41a2.png)
Finder with untouched AppKit file:
![screen shot 2018-11-02 at 7 23 28 pm](https://user-images.githubusercontent.com/44713744/47958318-47eb1300-df86-11e8-8145-34e22f8f12d1.png)
Hardware System Information:
![screen shot 2018-11-02 at 7 23 42 pm](https://user-images.githubusercontent.com/44713744/47958323-59ccb600-df86-11e8-8fa4-c3edc9202609.png)
Mid-2009 13" MBP 5,5 10.14.1 (18B75)
Without Reduced Transparency on, the light mode UI glitches are still apparent with no solid menu bar and grey sidebars.
EDIT: After uninstall and reinstall of the patches, no effects happen with or without transparency. Is there something I have to do to make the tweaked CoreUI or HIToolBox framework active?

**Comment by thegurgz** [comment]
I think I found the issue. When I uninstalled the patch the first time, I deleted the patched CoreUI and HIToolbox and the .bak files, leaving only the original HIToolbox or CoreUI file. I realize now that I should have deleted the original HIToolbox/CoreUI file along with the patched files and renamed the .bak files to the original HIToolbox/CoreUI. Since I didn't, the "original" HIToolbox/CoreUI files have been altered by the patch and now the patch will not work because the original framework is broken. I am not sure if this is correct, but it makes sense considering that the instructions say to make the .bak file and the troubleshooting is deleting the original file and restoring the .bak file. If I am correct, is there any way to obtain untouched CoreUI and HIToolbox files from someone else or am I SOL? I suppose a clean install will fix this...
